src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -999,7 +999,19 @@
      "CREATE INDEX idx_resource_events_resource_title ON resource_events(resource_title)"
      "CREATE INDEX idx_resource_events_status ON resource_events(status)"
      "CREATE INDEX idx_resource_events_timestamp ON resource_events(timestamp)"
-     "ALTER TABLE resource_events ADD CONSTRAINT resource_events_report_id_fkey FOREIGN KEY (report_id) REFERENCES reports(id) ON DELETE CASCADE"))
+     "ALTER TABLE resource_events ADD CONSTRAINT resource_events_report_id_fkey FOREIGN KEY (report_id) REFERENCES reports(id) ON DELETE CASCADE")
+
+    (sql/do-commands
+      "ALTER TABLE certnames ADD CONSTRAINT certnames_pkey PRIMARY KEY (name)"
+      "CREATE INDEX idx_certnames_latest_report_id ON certnames(latest_report_id)"
```

Comment:
We should be switching to the certnames_latest_report_id_idx format from now on. I just noticed we let this slip in the report_id changes though ...